### ISSUE TITLE
Elementor form field "Honeypot" hints attacker with what has gone wrong

### ISSUE THREAD

**Issue by ITaluone** [elementor/elementor#28480]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

If you add a form field of type "honeypot" a possible spambot gets a hint about why a form could not be sent.

Output in terminal when sending with curl:
```
{"success":false,"data":{"message":"Ihre Nachricht konnte nicht gesendet werden. ","errors":{"field_5aed512":"Ung\u00fcltiges Formular.","":""},"data":[]}}
```

This makes the honeypot _completely_ useless IMHO...

If I let my keys underneath my doormat and stick a post-it on the doorbell that the keys are underneath my doormat, it makes no sense to leave it there..

Can I somehow tell the form it should behave like "normal" but just send it?

### Steps to reproduce

1) Add a elementor form
2) Add a few fields, one of them a honepot
3) Open the Browser DevTools
4) Post a correct form
5) Copy the ajax.php as CURL command
6) Run curl command (before: add a text to the empty field from the honeypot)
7) execute
8) see response with hint about:
  - Failed
  - **even gives you a hint which field is invalid**

### Expected behavior

Behave like nothing is wrong: Just give the "normal" "Sent successful" message, but do not actually send..

### Isolating the problem

- [x] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [x] This bug happens with the Hello Elementor theme active.
- [x] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

```txt

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server - GPL v8.0.33
	PHP Version: 8.3.10
	PHP Memory Limit: 512M
	PHP Max Input Vars: 4000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.6.1
	Site URL: 
	Home URL: 
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: de_DE
	Timezone: Europe/Vienna
	Admin Email: thomas@tricksiebzehn.at
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 3.1.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: de-DE
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36

== Active Plugins ==
	Duplicator Pro
		Version: 4.5.18.2
		Author: Duplicator

	Dynamic.ooo - Dynamic Content for Elementor
		Version: 3.0.12
		Author: Dynamic.ooo

	Elementor
		Version: 3.23.4
		Author: Elementor.com

	Elementor Pro
		Version: 3.23.3
		Author: Elementor.com

	Image Optimizer by Elementor – Compress, Resize and Optimize Images
		Version: 1.5.1
		Author: Elementor.com

	JetElements For Elementor
		Version: 2.6.20.1
		Author: Crocoblock

	JetTricks
		Version: 1.4.9
		Author: Crocoblock

	Rank Math SEO
		Version: 1.0.226
		Author: Rank Math

	Rank Math SEO PRO
		Version: 3.0.60
		Author: Rank Math

	Redirection
		Version: 5.5.0
		Author: John Godley

	SVG Support
		Version: 2.5.8
		Author: Benbodhi

	WP Rocket
		Version: 3.16.4
		Author: WP Media


== Elemente Verwendung ==
	
	container : 3
		container : 12
		divider : 1
		heading : 5
		icon-box : 1
		icon-list : 3
		text-editor : 1
	footer : 1
		container : 4
		dce-views : 1
		heading : 2
		icon-list : 1
		nav-menu : 1
	header : 1
		container : 8
		icon : 1
		icon-list : 1
		image : 3
		nav-menu : 1
	popup : 4
		container : 21
		heading : 11
		icon-list : 8
		image : 1
		nav-menu : 1
		spacer : 3
		text-editor : 6
	section : 16
		button : 1
		container : 56
		dyncontel-title : 2
		heading : 23
		icon : 1
		icon-list : 20
		image : 3
		jet-inline-svg : 1
		social-icons : 1
		spacer : 1
		text-editor : 9
	wp-page : 10
		accordion : 1
		button : 3
		container : 194
		dce-views : 1
		divider : 6
		form : 1
		heading : 72
		icon : 3
		icon-box : 8
		icon-list : 19
		image : 2
		jet-inline-svg : 3
		loop-grid : 2
		nested-accordion : 1
		social-icons : 10
		spacer : 30
		text-editor : 29
		video : 3
	single-post : 1
		container : 1
		theme-post-content : 1
	loop-item : 3
		container : 12
		dyncontel-title : 1
		heading : 1
		image : 2
		theme-post-title : 1


== Einstellungen ==
	
	cpt_support: page, projekt, job
	allow_tracking: yes
	unfiltered_files_upload: 1
	disabled_elements: nested-tabs, inner-section, image-box, google_maps, image-carousel, image-gallery, counter, progress, testimonial, alert, audio, shortcode, html, sidebar, read-more, rating, wp-widget-pages, wp-widget-calendar, wp-widget-archives, wp-widget-media_audio, wp-widget-media_image, wp-widget-media_gallery, wp-widget-media_video, wp-widget-meta, wp-widget-search, wp-widget-text, wp-widget-categories, wp-widget-recent-posts, wp-widget-recent-comments, wp-widget-rss, wp-widget-tag_cloud, wp-widget-nav_menu, wp-widget-custom_html, wp-widget-block, jet-carousel, jet-banner, jet-dropbar, jet-horizontal-timeline, jet-slider, jet-hotspots, jet-unfold, jet-view-more, text-path, dyncontel-acfgallery, dyncontel-popup, dyncontel-template, dce-tokens, dce-discover-tokens, dce-dynamicposts-v2, dyncontel-content, dyncontel-excerpt, dyncontel-featured-image, dce-pdf-viewer, theme-site-logo, theme-site-title, theme-page-title, testimonial-carousel, media-carousel, call-to-action, flip-box, price-table, price-list, hotspot, animated-headline, slides, login, gallery, portfolio, posts, loop-carousel, archive-posts, theme-archive-title, theme-post-featured-image, theme-post-excerpt, nested-carousel, progress-tracker, stripe-button, paypal-button, video-playlist, code-highlight, lottie, template, facebook-page, facebook-embed, facebook-comments, facebook-button, blockquote, sitemap, post-info, post-comments, post-navigation, author-box, search-form, countdown, table-of-contents, reviews, pp-advanced-menu, pp-advanced-accordion, pp-testimonials


== Eigenschaften ==
	Custom Fonts: 2
	Custom Icons: 0

== Integrations ==
	


== Elementor-Experimente ==
	Verbessertes Laden von CSS: Aktiv
	Inline-Schriftarten-Symbole: Standardmäßig deaktiviert
	Additional Custom Breakpoints: Aktiv
	Flexbox Container: Aktiv
	Grid Container: Aktiv
	Upgrade Swiper Library: Inaktiv
	Nested Elements Performance: Standardmäßig aktiviert
	Optimized Control Loading: Standardmäßig deaktiviert
	Optimized Markup: Standardmäßig deaktiviert
	Plugin Onboarding: Standardmäßig aktiviert
	Default to New Theme Builder: Inaktiv
	Header und Footer: Inaktiv
	Elementor Home Screen: Standardmäßig aktiviert
	Editor Top Bar: Inaktiv
	Build with AI: Inaktiv
	Startseiten: Inaktiv
	Nested Elements: Aktiv
	Hintergrundbilder nachladen: Inaktiv
	Element Caching: Standardmäßig deaktiviert
	Link In Bio: Standardmäßig deaktiviert
	Floating Buttons: Standardmäßig deaktiviert
	Elementor Editor Events: Standardmäßig deaktiviert
	Bedingungen anzeigen: Inaktiv
	Form Submissions: Aktiv
	Menü: Inaktiv
	Taxonomy Filter: Inaktiv
	Suche: Standardmäßig deaktiviert


== Protokoll ==
	
Log: showing 20 of 422024-06-27 06:58:13 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.21.3',
  'to' => '3.22.1',
)]
2024-06-27 06:58:15 [info] elementor::elementor_updater Started 
2024-06-27 06:58:15 [info] Elementor/Upgrades - _on_each_version Start  
2024-06-27 06:58:16 [info] Elementor/Upgrades - _on_each_version Finished 
2024-06-27 06:58:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.8',
  'to' => '3.22.3',
)]
2024-06-27 06:58:19 [info] dynamic-content-for-elementor::dce_updater Started 
2024-06-27 06:58:19 [info] Dynamic.ooo - Dynamic Content for Elementor/Upgrades - _v_3_0_0_tokens_default_disabled Start  
2024-06-27 06:58:19 [info] Dynamic.ooo - Dynamic Content for Elementor/Upgrades - _v_3_0_0_tokens_default_disabled Finished 
2024-06-27 06:58:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Dynamic.ooo - Dynamic Content for Elementor',
  'from' => '3.0.3',
  'to' => '3.0.8',
)]
2024-08-08 15:53:49 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.22.3',
  'to' => '3.23.4',
)]
2024-08-08 15:53:49 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Dynamic.ooo - Dynamic Content for Elementor',
  'from' => '3.0.8',
  'to' => '3.0.11',
)]
2024-08-08 15:53:56 [info] elementor::elementor_updater Started 
2024-08-08 15:53:56 [info] Elementor/Upgrades - _on_each_version Start  
2024-08-08 15:53:56 [info] Elementor/Upgrades - _on_each_version Finished 
2024-08-08 15:53:56 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.22.3',
  'to' => '3.23.4',
)]
2024-08-08 15:54:04 [info] elementor-pro::elementor_pro_updater Started 
2024-08-08 15:54:04 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-08-08 15:54:04 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-08-08 15:54:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.22.1',
  'to' => '3.23.3',
)]
2024-08-22 08:47:57 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Dynamic.ooo - Dynamic Content for Elementor',
  'from' => '3.0.11',
  'to' => '3.0.12',
)]

JS: showing 1 of 1JS: 2024-06-03 09:12:35 [error X 3][https://domain.com/wp-includes/js/jquery/jquery.min.js?ver=3.7.1:2:28760] Cannot read properties of undefined (reading &#039;value&#039;) 



== Elementor - Compatibility Tag ==
	
	Dynamic.ooo - Dynamic Content for Elementor: Kompatibel
	Elementor Pro: Kompatibel
	Image Optimizer by Elementor – Compress, Resize and Optimize Images: Die Kompatibilität ist nicht angegeben
	JetElements For Elementor: Unkompatibel

== Elementor Pro - Compatibility Tag ==
	
	Dynamic.ooo - Dynamic Content for Elementor: Kompatibel
	JetElements For Elementor: Unkompatibel

```

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by KingYes** [comment]
Hi @ITaluone 
This bug will fix in the next release.

**Comment by ITaluone** [comment]
Thank you 👍

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.24.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
